src/App.js

```diff
@@ -1,7 +1,16 @@
 import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import { Login } from './pages/Login';
+import { Wallet } from './pages/Wallet';
```

Comment:
Alterando na `Login` e na `Wallet` p/ ser com o `export default`, a importação pode ser dessa forma.

```suggestion
import Login from './pages/Login';
import Wallet from './pages/Wallet';
```